### ISSUE TITLE
Time stamp does not update to actual time

### ISSUE THREAD

**Issue by MichiW1** [jomjol/AI-on-the-edge-device#230]
Great project, I like it a lot :-)
I have one issue with the time stamp. The device always reports a time stamp with 1970 within MQTT data, also after several days and with a lot of correct readings.
Is this a general problem? Do I have to configure NTP Service somewhere? 
Thanks for help, Michael

**Comment by jomjol** [comment]
By default the water meter tries to connect to ""pool.ntp.org" as a time server. If this is not working (e.g. no connection to external internet), you can specify a internal time server with:
```
[System]
...
TimeServer = fritz.box
...
```

Here for example the wlan router from AVM.

**Comment by MichiW1** [comment]
Thanks jomjol for your quick reply.
I got it up running now, but the issue was not due to an invalid timeserver.
The problem is, that I use the device with fixed IP-address and configured that in wifi.ini. By the given description, I did not enter any DNS while assuming that the device uses the gateway address as DNS. That does obviously not work, if I only enter the device-IP, gateway-IP and netmask, the device does not reach "pool.ntp.org".
As soon as I set DNS to gateway-IP address as well, I did not have any problems with timestamp anymore.
Maybe this is worth to add in the DNS description within wifi.ini ?
